includes/SiteConfiguration.php

```diff
@@ -42,12 +42,12 @@ class SiteConfiguration
     private $oauthLegacyConsumerTokens;
     private $oauthConsumerSecret;
     private $oauthIdentityGraceTime = '24 hours';
-    private $oauthMediaWikiCanonicalServer = 'http://en.wikipedia.org';
+    private $oauthMediaWikiCanonicalServer = 'https://en.wikipedia.org';
     private $xffTrustedHostsFile = '../TrustedXFF/trusted-hosts.txt';
     private $crossOriginResourceSharingHosts = array(
-        "http://en.wikipedia.org",
         "https://en.wikipedia.org",
-        "http://meta.wikimedia.org",
+        "https://en.wikipedia.org",
+        "https://meta.wikimedia.org",
```

Comment:
No need to duplicate these HTTPS urls - one instance is sufficient.

Reply:
Thankyou . I have updated. Removed duplicate Https url from SiteConfiguration.php .